helm-projectile.el

```diff
@@ -825,7 +833,7 @@ If it is nil, or ack/ack-grep not found then use default grep command."
      :default-directory default-directory
      :keymap helm-grep-map
      :history 'helm-grep-history
-     :truncate-lines t)))
+     :truncate-lines helm-projectile-truncate-lines)))
```

Comment:
I don't know if it is wise to have two variables dealing with truncation, one for `helm-grep` and another for `helm-projectile-grep-or-ack`.
How about use the customization variable provided by `helm-grep.el`: `helm-grep-truncate-lines` instead?